lib/src/types.ts

```diff
@@ -824,4 +826,5 @@ export class TypeTable {
 export interface TypeDef {
   type: Type;
   description?: string;
+  schemaProps?: SchemaProp[];
```

Comment:
Not all attributes in `SchemaProp` are applicable to each `Type`. Can we move the attributes to each individual type?

Reply:
Yes, good idea